src/myframe.h

```diff
@@ -21,9 +21,9 @@ struct MyFrame : wxFrame {
                  // device-independent pixels
     std::vector<std::string> scripts_in_menu;
     bool zenmode;
-    ColorDropdown *celldd = nullptr;

```

Comment:
Initialization directly in the declaration is superior to doing it in the initializer list when possible, so if anything, better to move other fields to the declaration where possible.

Reply:
Thanks for the hint! This saves a lot of lines.